toqito/matrix_props/tests/test_is_etf.py

```diff
@@ -0,0 +1,39 @@
+"""Test is_etf."""
+
+import numpy as np
+from toqito.matrix_ops import vectors_to_gram_matrix
+from toqito.matrix_props import is_etf
+
+def test_is_etf_valid_etf():
+    """Test that a known valid ETF matrix returns True."""
+    mat_valid_etf = np.array([
+        [1, -1/2, -1/2],
+        [0,  np.sqrt(3)/2, -np.sqrt(3)/2]
+    ])
+    np.testing.assert_equal(is_etf(mat_valid_etf), True)
+
+def test_is_etf_non_unit_norm_columns():
+    """Test that a matrix with non-unit norm columns returns False."""
+    mat_non_unit_norm = np.array([
+        [2, 1,  3],
+        [0, 2, -1]
+    ])
+    np.testing.assert_equal(is_etf(mat_non_unit_norm), False)
+
+def test_is_etf_unit_norm_but_not_equiangular():
+    """Test that a matrix whose columns have unit norm but are not equiangular returns False."""
+    # Each column has unit norm but the off-diagonal of the Gram matrix isn't constant.
+    mat_unit_but_not_equiangular = np.array([
+        [1,              0,           1/np.sqrt(2)],
```

Comment:
Don't add spacing as this is hard to maintain and is unnecessary for this matrix.